Warning: Hashable.hashValue is deprecated as a protocol requirement

I'm getting this warning when checking this out via SwiftPM:

```
.build/checkouts/swift-toml/Sources/Toml/Path.swift:41:16: warning: 'Hashable.hashValue' is deprecated as a protocol requirement; conform type 'Path' to 'Hashable' by implementing 'hash(into:)' instead
    public var hashValue: Int {
               ^
.build/checkouts/swift-toml/Sources/Toml/Tokens.swift:37:9: warning: 'Hashable.hashValue' is deprecated as a protocol requirement; conform type 'Token' to 'Hashable' by implementing 'hash(into:)' instead
    var hashValue: Int {
```

--- a/.gitignore
+++ b/.gitignore
@@ -2,3 +2,4 @@
 /.build
 /Packages
 /*.xcodeproj
+.swiftpm

--- a/Package.swift
+++ b/Package.swift
@@ -1,5 +1,4 @@
-// swift-tools-version:4.0
-// The swift-tools-version declares the minimum version of Swift required to build this package.
+// swift-tools-version:5.0
 
 import PackageDescription
 

--- a/Sources/Toml/Path.swift
+++ b/Sources/Toml/Path.swift
@@ -38,7 +38,11 @@ public struct Path: Hashable, Equatable {
         return true
     }
 
-    public var hashValue: Int {
+    public func hash(into hasher: inout Hasher) {
+        hasher.combine(componentsHashValue)
+    }
+
+    private var componentsHashValue: Int {
         return components.reduce(0, { $0 ^ $1.hashValue })
     }
 

--- a/Sources/Toml/Tokens.swift
+++ b/Sources/Toml/Tokens.swift
@@ -34,7 +34,11 @@ enum Token: Hashable {
     case TableEnd
     case Comment(String)
 
-    var hashValue: Int {
+    func hash(into hasher: inout Hasher) {
+        hasher.combine(caseHashValue)
+    }
+
+    private var caseHashValue: Int {
         switch self {
         case .Identifier:
             return 0

